examples/get_event_200.json

```
@@ -277,6 +277,33 @@
       "data": {
         "result": false
       }
+    },
+    "velocity": {
+      "data": {
+        "distinctIp": {
+          "intervals": {
+            "5m": 1,
+            "1h": 1,
+            "24h": 1
+          }
+        },
+        "distinctLinkedId": {
+        },
```

This seems mis-formatted 

```suggestion
        "distinctLinkedId": {}
```

---

You can commit this for all the 3

---

pushed